crates/walrus-service/src/node/events/event_blob_writer.rs

```diff
@@ -270,6 +271,66 @@ impl EventBlobWriterFactory {
         root_dir_path.join("event_blob_writer").join("blobs")
     }
 
+    /// Cleans up orphaned blob files from the filesystem that don't exist in any database.
+    fn cleanup_orphaned_blobs(
+        blobs_path: &Path,
+        pending: &DBMap<u64, PendingEventBlobMetadata>,
+        attested: &DBMap<(), AttestedEventBlobMetadata>,
+        failed_to_attest: &DBMap<(), FailedToAttestEventBlobMetadata>,
+        certified: &DBMap<(), CertifiedEventBlobMetadata>,
+    ) -> Result<()> {
+        let blobs = fs::read_dir(blobs_path)?;
+        for blob in blobs {
+            let blob_path = blob?.path();
+            if !blob_path.is_file() {
+                continue;
+            }
+            if blob_path.file_name() == Some(OsStr::new("current_blob")) {
```

Comment:
nit: maybe define all the const blob names (just 1 at the moment) as a `const &str` so that we won't overlook them.

Reply:
good call, i can handle this in a follow up.